1.8/usage/tutorials/dcos-101/cli.md

```diff
@@ -36,19 +36,19 @@ We will make further use of the CLI in the sections that follow.
 We have already encountered several DC/OS components (including Mesos, Marathon, or Metronome) while experimenting with the DC/OS CLI.
 But what other components make up DC/OS?
 
+## DC/OS Terminology
+We will focus on the most important terminology here. More details can be found in the [documentation](https://dcos.io/docs/1.8/overview/architecture/).
+* Master: Aggregates resource offers from all agent nodes and provides them to registered frameworks.
+* Agent: Runs a discrete Mesos task on behalf of a framework. The synonym of agent node is worker or slave node.
+* Scheduler: The scheduler component of a service, for example, the Marathon scheduler.
```

Comment:
This is a bit different than: https://dcos.io/docs/1.8/overview/concepts/#-a-name-dcos-scheduler-a-scheduler.